<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderCard</class>
 <widget class="QWidget" name="OrderCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>553</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="datetime">
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="orderCodeSymbol">
        <property name="text">
         <string>订单号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="orderCode">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="orderPaymentSymbol">
        <property name="text">
         <string>订单金额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="orderPayment">
        <property name="text">
         <string>金额</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="ReceiverSymbol">
          <property name="text">
           <string>收货人：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="RealName">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="DRName">
      <property name="text">
       <string>餐厅</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="OrderCondition">
      <property name="text">
       <string>完成状态</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>297</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="finish">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>去评论</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="receive">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>已收货</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="GoTPayBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>去支付</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>取消订单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
